study-builder/studies/brugada/study-events.conf

```diff
@@ -0,0 +1,435 @@
+      "dispatchToHousekeeping": true,
+      "order": 1
+    },
+
+    // BACKGROUND_INFORMATION reminder #2 - Final (6 weeks)
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "BACKGROUND_INFORMATION",
+        "statusType": "CREATED"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "templates": [
+          { "emailTemplate": ${emails.background_information_reminder_en}, "language": "en", "isDynamic": true }
+        ],
+        "linkedActivityCode": "BACKGROUND_INFORMATION",
+        "pdfAttachments": []
+      },
+      "cancelExpr": """
+        user.studies["brugada"].forms["BACKGROUND_INFORMATION"].isStatus("COMPLETE")
+      """,
+      "delaySeconds": ${delay.weeks.six},
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": true,
+      "order": 2
+    },
+
+    // GENETIC_TESTING reminder #1 (3 weeks)
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "GENETIC_TESTING",
+        "statusType": "CREATED"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "templates": [
+          { "emailTemplate": ${emails.genetic_testing_and_family_history_reminder_en}, "language": "en", "isDynamic": true }
+        ],
+        "linkedActivityCode": "GENETIC_TESTING",
+        "pdfAttachments": []
+      },
+      "cancelExpr": """
+        user.studies["brugada"].forms["GENETIC_TESTING"].isStatus("COMPLETE")
+      """,
+      "delaySeconds": ${delay.weeks.three},
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": true,
+      "order": 1
+    },
+
+    // GENETIC_TESTING reminder #2 - Final (6 weeks)
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "GENETIC_TESTING",
+        "statusType": "CREATED"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "templates": [
+          { "emailTemplate": ${emails.genetic_testing_and_family_history_reminder_en}, "language": "en", "isDynamic": true }
+        ]
+        "linkedActivityCode": "GENETIC_TESTING",
+        "pdfAttachments": []
+      },
+      "cancelExpr": """
+        user.studies["brugada"].forms["GENETIC_TESTING"].isStatus("COMPLETE")
+      """,
+      "delaySeconds": ${delay.weeks.six},
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": true,
+      "order": 2
+    },
+
+    // GENETIC_TESTING "Completion Thank You " email
+    {
+      "trigger": {
+        "type": "ACTIVITY_STATUS",
+        "activityCode": "GENETIC_TESTING",
+        "statusType": "COMPLETE"
+      },
+      "action": {
+        "type": "SENDGRID_EMAIL",
+        "templates": [
+          { "emailTemplate": ${emails.completion_thank_you_en}, "language": "en", "isDynamic": true }
+        ],
+        "pdfAttachments": []
+      },
+      "maxOccurrencesPerUser": 1,
+      "dispatchToHousekeeping": true,
+      "order": 1
+    },
+
+    // Activity Instance Creation
+    {
```

Comment:
@YuriyPavlovBI 
I am seeing only 1 ACTIVITY_INSTANCE_CREATION event in study-events.conf
Looks like while refactoring the conf code from origin/brugada to origin/study/brugada branch some stuff was missed. 
Guessing study in "dev" might have been deployed from origin/brugada branch.
I would recommend redeploying the study from this final branch and test it out to make sure nothing was missed.

Reply:
Also as part of redeploy to Dev, please do deploy of brugada tenant as well so that we can make sure tenant deploy using a0Deploy works when we initialize the study in other regions. 
Thansk

Reply:
No, origin/brugada abandoned and was never used. I'll remove that.
Regarding ACTIVITY_INSTANCE_CREATION event - there's the only one event because all the others are being created by Workflow Manager as we go based on workflow transitions.

Reply:
This activity is optional, so we have no transition to it and have to explicitly create

Reply:
<img width="1895" alt="image" src="https://user-images.githubusercontent.com/71830706/135489064-efa086fb-ee2c-4c5b-8dac-b542b4e8260d.png">